egs/wsj/s5/steps/segmentation/merge_targets_dirs.py

```diff
@@ -0,0 +1,128 @@
+#! /usr/bin/env python
```

Comment:
It is extremely complicated to escape string quotes and concatenate multiple rspecifier lines and pass to queue.pl in shell.